soc/arm/nxp_imx/rt5xx/soc.c

```diff
@@ -419,7 +419,8 @@ void imxrt_pre_init_display_interface(void)
 	 */
 	CLOCK_AttachClk(kAUX1_PLL_to_MIPI_DPHY_CLK);
 	CLOCK_InitSysPfd(kCLOCK_Pfd3,
-		DT_PROP(DT_NODELABEL(mipi_dsi), dphy_clk_div));
+		((528ull * 18ull * 1000000ull) /
```

Comment:
-1 to magic constants

Reply:
Noted- I'll drop these (or at least provide useful macros to document them) in next push